patches/chrome-installer-mac-signing-modification.py.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/chrome/installer/mac/signing/modification.py b/chrome/installer/mac/signing/modification.py
+index 62461b92268e722eaa091dbd4f5bdf1ba3f14585..f97dc9198881b19597100185cc681c7fc25284fc 100644
+--- a/chrome/installer/mac/signing/modification.py
++++ b/chrome/installer/mac/signing/modification.py
+@@ -63,8 +63,8 @@ def _modify_plists(paths, dist, config):
+         base_channel_tag_components = []
+         if base_tag:
+             base_channel_tag_components.append(base_tag)
+-        if dist.channel:
+-            base_channel_tag_components.append(dist.channel)
++        #if dist.channel:
++        #    base_channel_tag_components.append(dist.channel)
```

Comment:
Can't avoid this patching because this code overwrites our channel string of Info.plist here.